mmpose/datasets/datasets/top_down/topdown_onehand10k_dataset.py

```diff
@@ -0,0 +1,233 @@
+        1: 'thumb1',
+        2: 'thumb2',
+        3: 'thumb3',
+        4: 'thumb4',
+        5: 'forefinger1',
+        6: 'forefinger2',
+        7: 'forefinger3',
+        8: 'forefinger4',
+        9: 'middle_finger1',
+        10: 'middle_finger2',
+        11: 'middle_finger3',
+        12: 'middle_finger4',
+        13: 'ring_finger1',
+        14: 'ring_finger2',
+        15: 'ring_finger3',
+        16: 'ring_finger4',
+        17: 'pinky_finger1',
+        18: 'pinky_finger2',
+        19: 'pinky_finger3',
+        20: 'pinky_finger4'
+
+    Args:
+        ann_file (str): Path to the annotation file.
+        img_prefix (str): Path to a directory where images are held.
+            Default: None.
+        data_cfg (dict): config
+        pipeline (list[dict | callable]): A sequence of data transforms.
+        test_mode (bool): Store True when building test or
+            validation dataset. Default: False.
+    """
+
+    def __init__(self,
+                 ann_file,
+                 img_prefix,
+                 data_cfg,
+                 pipeline,
+                 test_mode=False):
+
+        super().__init__(
+            ann_file, img_prefix, data_cfg, pipeline, test_mode=test_mode)
+
+        self.ann_info['flip_pairs'] = []
+
+        self.ann_info['use_different_joints_weight'] = False
+        assert self.ann_info['num_joints'] == 21
+        self.ann_info['joints_weight'] = \
+            np.ones((self.ann_info['num_joints'], 1), dtype=np.float32)
+
+        self.db = self._get_db(ann_file)
+        self.image_set = set([x['image_file'] for x in self.db])
+        self.num_images = len(self.image_set)
+
+        print(f'=> num_images: {self.num_images}')
+        print(f'=> load {len(self.db)} samples')
+
+    def _get_db(self, ann_file):
+        """Load dataset."""
+        with open(ann_file, 'r') as f:
+            data = json.load(f)
+        tmpl = dict(
+            image_file=None,
+            center=None,
+            scale=None,
+            rotation=0,
+            joints_3d=None,
+            joints_3d_visible=None,
+            dataset='OneHand10K')
+
+        imid2info = {x['id']: x for x in data['images']}
+
+        num_joints = self.ann_info['num_joints']
+        gt_db = []
+
+        for anno in data['annotations']:
+            newitem = cp.deepcopy(tmpl)
+            image_id = anno['image_id']
+            newitem['image_file'] = os.path.join(
+                self.img_prefix, imid2info[image_id]['file_name'])
+
+            if max(anno['keypoints']) == 0:
+                continue
+
+            joints_3d = np.zeros((num_joints, 3), dtype=np.float)
+            joints_3d_visible = np.zeros((num_joints, 3), dtype=np.float)
+
+            for ipt in range(num_joints):
+                joints_3d[ipt, 0] = anno['keypoints'][ipt * 3 + 0]
+                joints_3d[ipt, 1] = anno['keypoints'][ipt * 3 + 1]
+                joints_3d[ipt, 2] = 0
+                t_vis = anno['keypoints'][ipt * 3 + 2]
```

Comment:
out of no reason, maybe write it as `t_vis = min(anno['keypoints'][ipt * 3 + 2], 1)`?